matrix-sdk-android/src/main/java/fr/gouv/tchap/android/sdk/api/session/identity/IdentityService.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.gouv.tchap.android.sdk.api.session.identity
+
+import fr.gouv.tchap.sdk.internal.session.identity.model.Platform
+import org.matrix.android.sdk.api.MatrixCallback
+import org.matrix.android.sdk.api.session.identity.ThreePid
+import org.matrix.android.sdk.api.session.terms.GetTermsResponse
+import org.matrix.android.sdk.api.session.terms.TermsService
+import org.matrix.android.sdk.api.util.Cancelable
+
+/**
+ * Provides access to services identity server can provide
+ */
+interface IdentityService {
```

Comment:
I think we should rename this class  to distinguish it from the [existing one](https://github.com/vector-im/element-android/blob/develop/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/identity/IdentityService.kt#L25) in matrix sdk.
I know we have our own package, so this is just a suggestion. Let's talk in real life about this

Reply:
I suggest "ThreePidPlatformResolver"